docs/reference/cat.asciidoc

```diff
@@ -115,7 +115,7 @@ case is the eighth one.
 
 [source,sh]
 --------------------------------------------------
-% curl '192.168.56.10:9200/_cat/indices?bytes=b' | sort -rnk8
+% curl '192.168.56.10:9200/_cat/indices?v&bytes=b&s=store.size:desc'
```

Comment:
Small comment: I think you either remove the v argument or you also need to update the following output as it does not contain the verbose output. Makes sense?